GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -385,7 +385,7 @@ Task ViewGitItemAsync(GitItem gitItem)
 
         private void tvGitTree_BeforeExpand(object sender, TreeViewCancelEventArgs e)
         {
-            if (e.Node.IsExpanded)
+            if (e.Node?.IsExpanded ?? false)
```

Comment:
Whilst technically `e.Node` can be `null`, I don't believe it is ever `null`.
https://github.com/dotnet/winforms/search?q=TreeViewCancelEventArgs

If anything line:393 should be fixed to remove `?`.

Reply:
Ok